include/bitcoin/bitcoin.hpp

```diff
@@ -112,6 +112,7 @@
 #include <bitcoin/bitcoin/utility/enable_shared_from_base.hpp>
 #include <bitcoin/bitcoin/utility/endian.hpp>
 #include <bitcoin/bitcoin/utility/exceptions.hpp>
+#include <bitcoin/bitcoin/utility/resource_lock.hpp>
```

Comment:
Sort order (also in Makefile.am). These are generated by libbitcoin-build using automatic sort, do it would get reordered on next generation.